apps/trading/components/order-list-container/order-list-container.tsx

```diff
@@ -0,0 +1,17 @@
+import { useOrders } from '../../hooks/use-orders';
+import { OrderList } from '@vegaprotocol/order-list';
+import { Splash } from '@vegaprotocol/ui-toolkit';
+
+export const OrderListContainer = () => {
+  const { orders, loading, error } = useOrders();
+
+  if (error) {
+    return <Splash>Something went wrong: {error.message}</Splash>;
+  }
+
+  if (loading) {
+    return <Splash>Loading...</Splash>;
+  }
+
+  return <OrderList orders={orders} />;
+};
```

Comment:
Can this be more generic? Async loader component that lives in UI toolkit that takes loading/error/children and just renders out the correct states rather than tying it directly to the order list container